### PR TITLE
[FrameworkBundle] remove the ChatterInterface alias when the chatter service is removed

### LINKED ISSUE #47122: You have requested a non-existent service "chatter"
### Symfony version(s) affected

5.4.10

### Description

When injecting (autowiring) Chatter Service in a controller method 
You have requested a non-existent service "chatter"

### How to reproduce

Install Notify component, create simple controller and try to inject ChatterInterface in a controller method

### Possible Solution

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #47122
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -2489,6 +2490,7 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             $container->getDefinition('chatter.transports')->setArgument(0, $config['chatter_transports']);
         } else {
             $container->removeDefinition('chatter');
+            $container->removeAlias(ChatterInterface::class);
         }
         if ($config['texter_transports']) {
             $container->getDefinition('texter.transports')->setArgument(0, $config['texter_transports']);
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php:2497]: Do we need to do the same here for the texter transports?

Comment (maxtor3569) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php:2497]: I think so i get the same error for texter

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php:2497]: indeed, done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**